package.json

```diff
@@ -33,6 +35,9 @@
         "prettier": "2.3.2",
         "rimraf": "^3.0.2",
         "salesforce-pageobjects": "^1.1.0",
+        "salesforceapp-pageobjects": "^236.1.1",
+        "ts-node": "^10.0.0",
+        "typescript": "^4.3.2",
```

Comment:
is there any reason, that we cannot use ts here? @olivier-martin-sf

Reply:
It's just to make things as simple as possible and put the emphasis on the page objects library and how to use it. The repo is small and quite straightforward so JS is a good candidate actually. Most of the TS benefits (`type checking`, inference, intellisense...) can also be applied to JS files (that's why we have a `jsconfig.json` file). 

On a side note, TS has actually some perf issues in type checking (I don't think that would be noticeable for a repo this size) and their support for some features around ES Modules is actually still very much WIP. I don't want to have to redirect customers to TS issues instead of focusing on what this repo is about.

Hopefully that gives some reasonable explanations